builder/src/main/java/cz/xtf/builder/db/AbstractDatabase.java

```diff
@@ -24,28 +24,42 @@ public abstract class AbstractDatabase extends DefaultStatefulAuxiliary {
 
     protected boolean withLivenessProbe;
     protected boolean withReadinessProbe;
+    protected boolean withStartupProbe;
 
     public AbstractDatabase(String symbolicName, String dataDir) {
         this("testuser", "testpwd", "testdb", symbolicName, dataDir);
     }
 
+    @Deprecated
```

Comment:
I'm thinking whether it's necessary to deprecate methods without startup probes?

Reply:
It's not necessary, but I find it clearer to keep the number of constructors at a minimum eg.
```
public AbstractDatabase(String symbolicName, String dataDir, boolean withLivenessProbe, boolean withReadinessProbe)
public AbstractDatabase(String symbolicName, String dataDir, boolean withLivenessProbe, boolean withReadinessProbe, boolean configureEnvironment)
public AbstractDatabase(String symbolicName, String dataDir, boolean withLivenessProbe, boolean withReadinessProbe, boolean withStartupProbe, boolean configureEnvironment)
```
could IMO be easily misused

Reply:
I see and agree that having less contructors is good. However we try to keep XTF backward compatible as much as possible. If there is no hard technical reason to deprecate them (for example in unprobable case when liveness probe would be removed from Openshift) we should keep them.